app/models/companies_import.rb

```diff
@@ -0,0 +1,67 @@
+class CompaniesImport
+  include ActiveModel::Model
+  require 'roo'
+
+  attr_accessor :file
+
+  def initialize(attributes = {})
+    attributes.nil? ? raise("File not selected!") : attributes.each { |name, value| send("#{name}=", value) }
+  end
+
+  def persisted?
+    false
+  end
```

Comment:
What do this method?

Reply:
This method tells rails that this object has no related table in our database.

Reply:
Ok